src/index.js

```diff
@@ -52,8 +51,8 @@ export default function createStore(state) {
 
 			// Note: perf tests verifying this implementation: https://esbench.com/bench/5a295e6299634800a0349500
 			return function() {
-				let args = [state];
-				for (let i=0; i<arguments.length; i++) args.push(arguments[i]);
+				let args = Array.prototype.slice(arguments);
```

Comment:
This should probably be `Array.prototype.slice.apply(arguments)` because currenty the call will return just an empty array.
But afaik you should *not* use Array#slice to convert an arguments object to an array.
This leaks the `arguments` object an therefore can have a fatal performance impact, depending on the js engine.
But I haven't profiled that within the last 6ish months.